test/metabase/query_processor_test/remapping_test.clj

```diff
@@ -320,3 +320,37 @@
                             [2 123 "Mediocre Wooden Bench"  "Mediocre Wooden Bench"]
                             [3 105 "Fantastic Wool Shirt"   "Fantastic Wool Shirt"]]
                            (mt/rows (qp/process-query q3))))))))))))))
+
+(defmacro with-card-with-query
```

Comment:
@metamben Thanks for pointing that out. I added a test to reproduce the bug here.